orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/conditions/EvaluateConditionTask.java

```diff
@@ -120,6 +127,13 @@ public TaskResult execute(@Nonnull Stage stage) {
     }
   }
 
+  private void recordDeployPause(WaitForConditionContext ctx) {
+    registry.counter(
```

Comment:
@dreynaud wondering if a gauge is more appropriate

Reply:
lgtm like this with a counter! 👍